PopPUNK/citation.py

```diff
@@ -92,27 +94,34 @@ def print_citation(args, assign=False):
                                  "poppunk_version" : __version__,
                                  "poppunk_doi" : "10.1101/gr.241455.118"})
     else:
+
         if args.fit_model == "bgmm":
             model_mode = "BGMM"
             model_options = str(args.K) + " components"
+            cluster_type = "strains"
```

Comment:
I like this addition

Reply:
I think it's clearer, we'll see what happens in practice.